troubleshoot/ingest/opentelemetry/edot-collector/metadata.md

```diff
@@ -0,0 +1,81 @@
+---
+navigation_title: Collector doesn’t propagate metadata
+description: Learn why the Collector doesn’t extract custom attributes and how to propagate such values using EDOT SDKs.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# EDOT Collector doesn’t propagate client metadata
+
+If your EDOT Collector doesn’t propagate metadata like `project_id`, `tenant`, or `environment`, you may be misunderstanding what “client metadata” means in Elastic Distributions of OpenTelemetry (EDOT). 
```

Comment:
The current wording implies user error (“you may be misunderstanding”). Could we reframe this to highlight the Collector’s behavior instead? That keeps the tone neutral and more helpful.

Reply:
I like the latter option. I would lead with the default though:

> By default, the Collector only propagates transport-level metadata. If you want the EDOT Collector to propagate metadata like `project_id`, `tenant`, or `environment`, you need a specific SDK instrumentation.